presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManagerStats.java

```diff
@@ -111,6 +121,12 @@ public long getRunningQueries()
     }
 
     @Managed
+    public long getQueuedQueries()
```

Comment:
I had a question about this, from what I see CounterStat will have a decay function, in this case, I want to check at any given point how many queries are queued (an exact number, similar to running queries).